README.md

```diff
@@ -158,43 +172,55 @@ pip install pytest pyyaml pytest-instafail
 
 **Troubleshooting**
 
-If a `ModuleNotFoundError: No module named ‘torchrl._torchrl` errors occurs, 
-it means that the C++ extensions were not installed or not found. 
-One common reason might be that you are trying to import torchrl from within the 
-git repo location. Indeed the following code snippet should return an error if 
+If a `ModuleNotFoundError: No module named ‘torchrl._torchrl` errors occurs,
+it means that the C++ extensions were not installed or not found.
+One common reason might be that you are trying to import torchrl from within the
+git repo location. Indeed the following code snippet should return an error if
 torchrl has not been installed in `develop` mode:
+
 ```
 cd ~/path/to/rl/repo
 python -c 'from torchrl.envs import GymEnv'
 ```
+
 If this is the case, consider executing torchrl from another location.
 
-On **MacOs**, we recommend installing XCode first. 
-With Apple Silicon M1 chips, make sure you are using the arm64-built python 
+On **MacOs**, we recommend installing XCode first.
+With Apple Silicon M1 chips, make sure you are using the arm64-built python
 (e.g. [here](https://betterprogramming.pub/how-to-install-pytorch-on-apple-m1-series-512b3ad9bc6)). Running the following lines of code
 
 ```
 wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
 python collect_env.py
 ```
+
 should display
+
 ```
 OS: macOS *** (arm64)
 ```
+
 and not
+
 ```
 OS: macOS **** (x86_64)
 ```
 
+We have documented some [common issues (and workarounds)](COMMON_MUJOCO_ISSUES.md) when rendering Mujoco environments.
```

Comment:
Note to [myself](@vmoens): if we decide to rename / move that file we have to change this